docs/versioned_docs/version-2.11.0/stateful-applications/cassandra.md

```diff
@@ -0,0 +1,240 @@
+  ```
+
+  Example command:
+
+  ```
+  cassandra@cassandra-openebs-node-0:/$ cqlsh cassandra-openebs-svc.cassandra.svc.cluster.local
+  
+  Connected to cassandra-openebs at cassandra-openebs-svc.cassandra.svc.cluster.local:9042.
+  [cqlsh 5.0.1 | Cassandra 3.11.6 | CQL spec 3.4.4 | Native protocol v4]
+  Use HELP for help.
+  cqlsh>
+  ```
+
+- Creating a Keyspace. Now, let’s create a Keyspace and add a table with some entries into it.
+
+  ```
+  cqlsh> create keyspace dev
+  ... with replication = {'class':'SimpleStrategy','replication_factor':1};
+  
+- Creating Data Objects
+
+  ```
+  cqlsh> use dev;
+  cqlsh:dev> create table emp (empid int primary key,
+  ... emp_first varchar, emp_last varchar, emp_dept varchar);
+  
+- Inserting and Querying Data
+
+  ```
+  $ cqlsh:dev> insert into emp (empid, emp_first, emp_last, emp_dept)
+  ... values (1,'fred','smith','eng');
+
+  $ cqlsh:dev> select * from emp;
+  empid | emp_dept | emp_first | emp_last
+  -------+----------+-----------+----------
+      1 |      eng |      fred |    smith
+  (1 rows)
+
+- Updating a data
+
+  ```
+  $ cqlsh:dev> update emp set emp_dept = 'fin' where empid = 1;
+  $ cqlsh:dev> select * from emp;
+  empid | emp_dept | emp_first | emp_last
+  -------+----------+-----------+----------
+      1 |      fin |      fred |    smith
+  (1 rows)
+  cqlsh:dev> exit
+  ```  
+### Testing Cassandra Performance on OpenEBS
+   
+- Login to one of the cassandra pod and run the following sample loadgen command to write and read some entry to and from the database.
+
+  ```
+  $ kubectl exec -it cassandra-openebs-node-0 bash -n cassandra
+  ```
+
+- Get the database health status
+
+  ```
+  $ nodetool status
+  Datacenter: datacenter1
+  =======================
+  Status=Up/Down
+  |/ State=Normal/Leaving/Joining/Moving   --   Address        Load       Tokens       Owns (effective)  Host ID                               Rack
+  UN  192.168.52.94  135.39 MiB  256          32.6%             68206664-b1e7-4e73-9677-14119536e42d  rack1
+  UN  192.168.7.79   189.98 MiB  256          36.3%             5f6176f5-c47f-4d12-bd16-c9427baf68a0  rack1
+  UN  192.168.70.87  127.46 MiB  256          31.2%             da31ba66-42dd-4c85-a212-a0cb828bbefb  rack1
+  ``` 
+
+- Go to the directory where the binary is located.
+
+  ```
+  cassandra@cassandra-openebs-node-0:/$ cd /opt/cassandra/tools/bin
+  ```
+
+- Run Write load 
+
+  ```
+  cassandra@cassandra-openebs-node-0:/opt/cassandra/tools/bin$ ./cassandra-stress write n=1000000 -rate threads=50 -node 192.168.52.94
+  ```
+
+- Run Read Load
+
+  ```
+  cassandra@cassandra-openebs-node-0:/opt/cassandra/tools/bin$ ./cassandra-stress read n=200000 -rate threads=50 -node 192.168.52.94
+  ```
+## See Also:
+
+[OpenEBS architecture](/docs/concepts/architecture)
+
+[OpenEBS use cases](/docs/introduction/usecases)
+
+[Local PV concepts](/docs/concepts/localpv)
+
+[Understanding NDM](/docs/user_guides/ugndm)
```

Comment:
all the links in see also should be in a single line, just keep the spaces bw links

Reply:
![image](https://user-images.githubusercontent.com/75615087/127900970-e03dd714-f49b-44ab-8e91-fc5e35776446.png)

Reply:
Done